rocket-chat-android-widgets/src/main/java/chat/rocket/android/widget/RocketChatWidgets.java

```diff
@@ -26,7 +31,7 @@ public static void initialize(Context context, OkHttpClient okHttpClient) {
     ImagePipelineConfig imagePipelineConfig = OkHttpImagePipelineConfigFactory
         .newBuilder(context, okHttpClient)
         .setRequestListeners(listeners)
-        .setImageDecoderConfig(CustomImageFormatConfigurator.createImageDecoderConfig())
+//        .setImageDecoderConfig(CustomImageFormatConfigurator.createImageDecoderConfig())
```

Comment:
If we need to plan to support SVG image in the future I think that we could leave this commented code. If not, we can remove it (and the CustomImageFormatConfigurator and SvgDecoder class too).